researchspace/researchspace
# Add extra files to buildZip for digilib configuration

# Why

The zip bundle contains digilib. The default templates are written for cantaloupe, and the uris for images would be broken.

# What

ui.prop and IIIFConfig template
global.prop with loadDefaultConfig set to 1

# How To Test

./gradlew clean buildZip

cd build/output/researchspace

./start.sh

Uploading and viewing images should work.
